### DISCUSSION TITLE
Reversing the Mouse Roller Action for Zooming

### DISCUSSION THREAD

**Discussion by nebsar** [pelicanmapping/osgearth#1940] [Support Q&A] [answered]
Is it possible to reverse the mouse roller action for zooming in and out? If so, how can I do it?

**Comment by keince** [comment, marked as answer]
Yes.

EarthManipulator* em = new EarthManipulator();
em->getSettings()->bindScroll(EarthManipulator::ACTION_ZOOM_IN, osgGA::GUIEventAdapter::SCROLL_UP);
em->getSettings()->bindScroll(EarthManipulator::ACTION_ZOOM_OUT, osgGA::GUIEventAdapter::SCROLL_DOWN);

-S

**Reply by nebsar** [reply]
Thanks keince.
